public/static/docs/command-reference/commit.md

```diff
@@ -21,7 +21,13 @@ The `dvc commit` command is useful for several scenarios where a dataset is
 being changed: when a [stage](/doc/command-reference/run) or
 [pipeline](/doc/command-reference/pipeline) is in development, when one wishes
 to run commands outside the control of DVC, or to force DVC-file updates to save
-time tying stages or a pipeline.
+time tying stages or a pipeline. These scenarios are further detailed below.
+
+💡 For convenience, a Git hook is available to remind you to `dvc commit` when
+needed after a `git commit`. See `dvc install` for more details.
+
+> Note that the hooks only have an effect in repository versions (branches,
```

Comment:
probably too much to include this note here

Reply:
Removing...